File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/ConferenceUpdated.cs
using System;
using ENode.Eventing;

namespace ConferenceManagement
{
    public class ConferenceUpdated : DomainEvent<Guid>
    {
        public ConferenceEditableInfo Info { get; private set; }

        public ConferenceUpdated() { }
        public ConferenceUpdated(ConferenceEditableInfo info)
        {
            Info = info;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/events/ConferenceUpdated.java
package com.microsoft.conference.management.domain.events;

import com.microsoft.conference.management.domain.models.ConferenceEditableInfo;
import org.enodeframework.eventing.DomainEvent;

public class ConferenceUpdated extends DomainEvent<String> {
    private ConferenceEditableInfo info;

    public ConferenceUpdated() {
    }

    public ConferenceUpdated(ConferenceEditableInfo info) {
        this.info = info;
    }

    public ConferenceEditableInfo getInfo() {
        return info;
    }

    public void setInfo(ConferenceEditableInfo info) {
        this.info = info;
    }
}
